<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>651</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1" stretch="1,1">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <column>
         <property name="text">
          <string>mail</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>cognome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>data</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>sesso</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QChartView" name="graphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>Lista Utenti</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>Percentuale Uomini/Donne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>Percentuale eta'</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
